### PR TITLE
orders-controller

### PR DESCRIPTION
**Pull request by Nerfi**
check out the orders controller

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  get 'order/new'
```
Comment (danielppolo) [inline config/routes.rb]: Here you need to add the method you will execute for each route

File: test/controllers/orders_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
```
Comment (danielppolo) [inline test/controllers/orders_controller_test.rb]: All the code of the orders controller shouldn't be here, this is orders_controller_TEST.rb, it should be in orders_controller.rb

File: test/controllers/orders_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+class OrdersControllerTest < ActionDispatch::IntegrationTest
+
+def new
+  @order = Order.new
+  authorize @order
```
Comment (danielppolo) [inline test/controllers/orders_controller_test.rb]: We won't use Pundit, so we will not add any authorize in the code

File: test/controllers/orders_controller_test.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  @meal = Meal.find(params[:meal_id])
+
+end
+
+def create
+  @order = Order.new(order_params)
```
Comment (danielppolo) [inline test/controllers/orders_controller_test.rb]: You are using the order_params, but that means you have created a method with the same name. In kitt you can find it as Strong Params

File: test/controllers/orders_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  @order.meal = Meal.find(params[:meal_id])
+  @order.user = current_user
+  authorize @order
```
Comment (danielppolo) [inline test/controllers/orders_controller_test.rb]: No authorize

File: test/controllers/orders_controller_test.rb (hunk 2 above)
Comment (Nerfi) [reply test/controllers/orders_controller_test.rb]: done

File: test/controllers/orders_controller_test.rb (hunk 3 above)
Comment (Nerfi) [reply test/controllers/orders_controller_test.rb]: done

**Pull request merged**